app/src/main/res/layout/fragment_plant_detail.xml

```diff
@@ -83,21 +76,35 @@
                 android:layout_height="match_parent">
 
                 <TextView
-                    android:id="@+id/plant_watering"
+                    android:id="@+id/plant_name"
+                    style="?android:attr/textAppearanceLarge"
```

Comment:
Using android:textAppearance would be more suitable than using style as this attr only affects text display